config/initializers/application_controller_renderer.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+# Be sure to restart your server when you modify this file.
+
+# ActiveSupport::Reloader.to_prepare do
+#   ApplicationController.renderer.defaults.merge!(
+#     http_host: 'example.org',
+#     https: false
+#   )
+# end
```

Comment:
Does this file need to exist if it has nothing in it but comments?  Maybe Rails gets upset without it?